Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG TERRAFORM_VERSION=1.1.6
+ARG TERRAFORM_VERSION=1.0.0
```

Comment:
If this is being changed, is the doc being updated as well?

Reply:
No longer applicable